stickynotes/stickynotes_applet_callbacks.c

```diff
@@ -108,452 +105,537 @@ applet_key_cb (GtkWidget         *widget,
+                                                   "MATE Desktop Environment"),
+                           "authors",            authors,
+                           "documenters",        documenters,
+                           "translator-credits", _("translator-credits"),
+                           "logo-icon-name",     "mate-sticky-notes-applet",
+                           NULL);
 }
 
 /* Preferences Callback : Save. */
 void
 preferences_save_cb (gpointer data)
 {
-	gint width = gtk_adjustment_get_value (stickynotes->w_prefs_width);
-	gint height = gtk_adjustment_get_value (stickynotes->w_prefs_height);
-	gboolean sys_color = gtk_toggle_button_get_active (
-			GTK_TOGGLE_BUTTON (stickynotes->w_prefs_sys_color));
-	gboolean sys_font = gtk_toggle_button_get_active (
-			GTK_TOGGLE_BUTTON (stickynotes->w_prefs_sys_font));
-	gboolean sticky = gtk_toggle_button_get_active (
-			GTK_TOGGLE_BUTTON (stickynotes->w_prefs_sticky));
-	gboolean force_default = gtk_toggle_button_get_active (
-			GTK_TOGGLE_BUTTON (stickynotes->w_prefs_force));
-	gboolean desktop_hide = gtk_toggle_button_get_active (
-			GTK_TOGGLE_BUTTON (stickynotes->w_prefs_desktop));
-
-	if (g_settings_is_writable (stickynotes->settings,"default-width"))
-		g_settings_set_int (stickynotes->settings,"default-width", width);
-	if (g_settings_is_writable (stickynotes->settings,"default-height"))
-		g_settings_set_int (stickynotes->settings,"default-height", height);
-	if (g_settings_is_writable (stickynotes->settings,"use-system-color"))
-		g_settings_set_boolean (stickynotes->settings,"use-system-color", sys_color);
-	if (g_settings_is_writable (stickynotes->settings,"use-system-font"))
-		g_settings_set_boolean (stickynotes->settings,"use-system-font", sys_font);
-	if (g_settings_is_writable (stickynotes->settings,"sticky"))
-		g_settings_set_boolean (stickynotes->settings,"sticky", sticky);
-	if (g_settings_is_writable (stickynotes->settings,"force-default"))
-		g_settings_set_boolean (stickynotes->settings,"force-default", force_default);
-	if (g_settings_is_writable (stickynotes->settings,"desktop-hide"))
-		g_settings_set_boolean (stickynotes->settings,"desktop-hide", desktop_hide);
+    gint width = gtk_adjustment_get_value (stickynotes->w_prefs_width);
+    gint height = gtk_adjustment_get_value (stickynotes->w_prefs_height);
+    gboolean sys_color = gtk_toggle_button_get_active (
+        GTK_TOGGLE_BUTTON (stickynotes->w_prefs_sys_color));
+    gboolean sys_font = gtk_toggle_button_get_active (
+        GTK_TOGGLE_BUTTON (stickynotes->w_prefs_sys_font));
+    gboolean sticky = gtk_toggle_button_get_active (
+        GTK_TOGGLE_BUTTON (stickynotes->w_prefs_sticky));
+    gboolean force_default = gtk_toggle_button_get_active (
+        GTK_TOGGLE_BUTTON (stickynotes->w_prefs_force));
+    gboolean desktop_hide = gtk_toggle_button_get_active (
+        GTK_TOGGLE_BUTTON (stickynotes->w_prefs_desktop));
```

Comment:
you don’t have to leave an opening parenthesis alone

Reply:
As i said code-style was horror :)
I will break the line after `=` sign